chapters/testing.Rmd

```diff
@@ -718,21 +753,20 @@ $ coverage report -m
 ```
 
 ```text
-Name            Stmts   Miss  Cover   Missing
----------------------------------------------
-countwords.py      20      8    60%   19-21, 25-31
-plotcounts.py      46     27    41%   41-47, 67-69, 74-90, 94-104
-test_zipfs.py      32      0   100%
-utilities.py        7      4    43%   24-27
----------------------------------------------
-TOTAL             105     39    63%
+bin/countwords.py      20      7    65%   25-26, 30-38
+bin/plotcounts.py      58     37    36%   48-55, 75-77, 82-83, 
+                                          88-118, 122-140
+bin/test_zipfs.py      31      0   100%
+bin/utilities.py        8      5    38%   18-22
+-------------------------------------------------
+TOTAL                 117     49    58%
```

Comment:
The figure corresponding to this coverage report (`figures/testing/python-coverage.png`) used to have a white background, while the new one had a black background. I'm wondering if in the printed book a white background would look better? (Don't worry if it's not easy for you to regenerate the new figure with a white background.)

Reply:
Done! Thanks for reminding me; I had to switch my OS preference back from dark (I prefer my computer to reflect my soul)